### PR TITLE
Update FIPs 26 to include new API endpoint

### PR DESCRIPTION
**Pull request by tvl83**

### CODE REVIEW

File: fip-0026.md [hunk 1]
```diff
@@ -103,7 +107,73 @@ Holds information related to the marketplace.
...
+|listings|status|Int|Status of the listing.|
+| |more|Int|Number of results remaining|
+##### Example
+```
+{
+  "listings": [
+    {
+        "commission_fee": "6.00000000000000000",
+        "date_listed": 1638229065,
+        "date_updated": 1638229065,
```
Comment (pawmmm) [inline fip-0026.md:163]: We should follow convention and return time as "2020-09-11T18:30:56"

File: fip-0026.md [hunk 2]
```diff
@@ -103,7 +107,73 @@ Holds information related to the marketplace.
...
+        "domain": "zfoywvvftd",
+        "domainhash": "0x47a2e16e6b54a219aaaa16cc9430284d",
+        "owner": "wjeo4abnk4c2",
+        "ownerhash": "0x24e09545787058055ad8fa0a3710c33d",
```
Comment (pawmmm) [inline fip-0026.md:167]: Why do we need the hashes?

**Review by pawmmm — CHANGES_REQUESTED** [review]

File: fip-0026.md (hunk 2 above)
Comment (tvl83) [reply fip-0026.md:167]: I can remove them.

File: fip-0026.md (hunk 1 above)
Comment (tvl83) [reply fip-0026.md:163]: okay. I just found an example on how to create that date format from the timestamp, so I'll update the C++ code and the FIPs shortly.

**Review by pawmmm — APPROVED** [review]

**Pull request merged**